pom.xml

```diff
@@ -419,7 +419,7 @@
           <dependency>
             <groupId>com.puppycrawl.tools</groupId>
             <artifactId>checkstyle</artifactId>
-            <version>8.45.1</version>
+            <version>10.21.0</version>
```

Comment:
Our build is running on java 8, check style 10+ require Java 11 - I think this will not work